### DISCUSSION TITLE
There is no official release tag of `2.0.0`

### DISCUSSION THREAD

**Discussion by zhuoqun-chen** [dfki-ric/phobos#305] [Q&A] [answered]
Hello developers,

It seems that I can’t find any official release tag of `2.0.0` version, I can only see there’s a branch called `pre_2.0.0`. Could you please tag it so people like me can download the `phobos.zip` more easily?

Thank you very much!

**Comment by hwiedPro** [comment, marked as answer]
Yep, we are currently fixing some final bugs. I'm planning the release for end of august
